libs/datasets/can_model_output_schema.py

```diff
@@ -1,29 +1,59 @@
 """
 Ouput of the 
 """
+DATE = "date"
+TOTAL = "total"
+SUSCEPTIBLE = "susceptible"
+EXPOSED = "exposed"
+INFECTED = "infected"
+INFECTED_A = "infected_a"
+INFECTED_B = "infected_b"
+INFECTED_C = "infected_c"
+ALL_HOSPITALIZED = "all_hospitalized"
+ALL_INFECTED = "all_infected"
+DEAD = "dead"
+BEDS = "beds"
+POPULATION = "population"
+
+#,date,
+# total,
+# susceptible,
+# exposed,
+# infected,
+# infected_a,
+# infected_b,
+# infected_c,
+# recovered,
+# dead,
+# pct_change,
+# doubling_time,
+# beds,
+# all_hospitalized,
+# all_infected
+
 CAN_MODEL_OUTPUT_SCHEMA = [
     "day_num",
     # ^ for index column
-    "date",
-    "a",
-    "b",
-    "c",
-    "d",
-    "e",
-    "f",
-    "g",
-    "all_hospitalized",
-    "all_infected",
-    "dead",
-    "beds",
+    DATE,
+    TOTAL,
+    SUSCEPTIBLE,
+    EXPOSED,
+    INFECTED, 
+    INFECTED_A, # (not hospitalized, but infected)
+    INFECTED_B, # infected_b (hospitalized not in icu)
+    INFECTED_C, # infected_c (in icu)
+    ALL_HOSPITALIZED, # infected_b + infected_c
+    ALL_INFECTED, # infected_a + infected_b + infected_c
+    DEAD,
+    BEDS,
     "i",
     "j",
     "k",
     "l",
-    "population",
+    POPULATION,
     "m",
     "n",
```

Comment:
Generally confused about what these are? someone pointed me to a few places and i looked at the original file, but where lines 18-32 came from

Reply:
they are just unused values from old spreadsheet